sw/host/tests/chip/flash_ctrl/src/main.rs

```diff
@@ -0,0 +1,293 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use clap::Parser;
+
+use std::iter;
+use anyhow::Context;
+use opentitanlib::test_utils::lc_transition;
+use opentitanlib::chip::boolean::MultiBitBool8;
+use opentitanlib::dif::lc_ctrl::{
+    DifLcCtrlState, DifLcCtrlToken, LcCtrlReg, LcCtrlStatus, LcCtrlTransitionCmd,
+    LcCtrlTransitionRegwen,
+};
+
+use std::collections::HashMap;
+use std::fs;
+use std::path::PathBuf;
+use std::time::Duration;
+
+use object::{Object, ObjectSymbol};
+use opentitanlib::app::TransportWrapper;
+use opentitanlib::execute_test;
+use opentitanlib::test_utils::init::InitializeTest;
+use opentitanlib::test_utils::mem::{MemRead32Req, MemWrite32Req};
+use opentitanlib::uart::console::UartConsole;
+use opentitanlib::io::jtag::JtagTap;
+use opentitanlib::test_utils::load_sram_program::{ExecutionMode, SramProgramParams
+};
+
+#[derive(Debug, Parser)]
+struct Opts {
+    #[command(flatten)]
+    init: InitializeTest,
+
+    /// Console receive timeout.
+    #[arg(long, value_parser = humantime::parse_duration, default_value = "600s")]
+    timeout: Duration,
+
+    /// Path to the firmware's ELF file, for querying symbol addresses.
+    #[arg(value_name = "FIRMWARE_ELF")]
+    firmware_elf: PathBuf,
```

Comment:
`SramProgramParams` already contains a path to the left (it's an `Option` but you know you want an ELF) so I think you could remove this parameter and use the one in `sram_program`. We really need to remove support for vmem loading for sram programs anyway.

Reply:
> `SramProgramParams` already contains a path to the left (it's an `Option` but you know you want an ELF) so I think you could remove this parameter and use the one in `sram_program`. We really need to remove support for vmem loading for sram programs anyway.

With sram_program, how can I change line 263 and 264? I am rust newbie :)